api/pom.xml

```diff
@@ -206,6 +206,9 @@
                             <generateApiTests>false</generateApiTests>
                             <templateDirectory>src/main/resources/custom_templates</templateDirectory>
                             <library>apache-httpclient</library>
+                            <configOptions>
+                                <useRuntimeException>true</useRuntimeException>
```

Comment:
@clementdenis I can't find this documented at https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator-maven-plugin/README.md 🤔

Reply:
This is a generator-specific option, so it's not documented directly in the maven plugin.

But it's listed in the options of the java generator: https://openapi-generator.tech/docs/generators/java/